src/settings.py

```diff
@@ -80,6 +80,8 @@
 LOVER_WINS_WITH_FOOL = False # if fool is lynched, does their lover win with them?
 DEFAULT_SEEN_AS_VILL = True # non-wolves are seen as villager regardless of the default role
 
+START_VOTES_REQUIRED = lambda num_players: min((num_players + 1) / 2.2, 5)
```

Comment:
If it's a function (even a lambda), make it lowercase, and preferably in the functions section. This will, among other things, allow us to segregate settings from functions in the future.